Utils/src/cz/fidentis/utils/MeshUtils.java

```diff
@@ -8,7 +8,9 @@
 
 
 import cz.fidentis.model.Model;
+import java.util.ArrayList;
 import java.util.List;
+import javafx.geometry.Point3D;

```

Comment:
Myslim ze nie je moc vhodne pridavat zavislost na dalsiu triedu ktora reprezentuje bod. Bolo by dobre keby mame zjednotene to s cim pracuju nase algoritmy, cize napriklad aby vsade kde sa pracuje s bodom sa k tomu pouzivala ta ista trieda. Viem ze to v nasom kode vobec neplati, ale bolo by fajn keby aspon novy kod dodrzoval nejake pravidla atd, cize si myslim ze by bolo lepsie keby `giftWrapping` pracoval s `List<Vector3f>` namiesto `Point3D`. Vdaka tomu by sa napriklad jednoducho a hned dalo pouzit tento algoritmus na cely model atd.
@ZueFe myslis ze sa to hodi menit?

Chapem ale ze pravdepodobne cely tvoj kod pracuje s `Point3D` coz znamena ze by to vyzadovalo velmi vela zmien keby si to chcel upravit takze necakam ze to teraz cele prerobis :D Kazdopadne by bolo fajn keby si mohol aspon veci v spolocnom kode (Cize napriklad tu v MeshUtils) spravit tak aby pracovali s `Vector3f`
Ak to bude vela prace alebo z nejakeho dovodu komplikovane tak to nechaj tak, ale ja myslim ze by to malo byt celkom lahko upravitelne.

Reply:
Ked na to tak v rychlosti pozeram, tak Vector3f ma rovnake metody ako Point3D, takze by to nemusel byt az taky problem zmenit.

Reply:
hotovo, nakomitujem to ako celok